Dockerfile

```diff
@@ -27,7 +27,7 @@ RUN pip install -r requirements.txt
 
 # Fetch UI code
 COPY Makefile .
-ARG UI_RELEASE=0.9.3
+ARG UI_RELEASE=latest
```

Comment:
@gtanzillo the script is defaulting to latest, but we could be specific here if we prefer so.

Reply:
Sounds good, we may need to lock to s specific version in the future, but now latest is good